_i18n/fr/resources/moneropedia/cryptocurrency.md

```diff
@@ -1,22 +1,21 @@
 
-### More Information
+### Plus d'Informations
 
-Cryptocurrency is the generic term for a large set of digital assets that use encryption techniques to generate units of currency, verify the transactions, and transfer value. Generally, cryptocurrencies are considered to be decentralized. Cryptocurrency should not be confused with virtual currency which is a type of digital money that is usually controlled by its creators or developers. Some examples of virtual currency are gametime in World of Warcraft, ROBUX in Roblox, reward points programs, or Ripple, all of which can be exchanged for currency or cash value, but are not considered cryptocurrency because they are centalized and controlled/issued by a single entity.  
+Cryptomonnaie est le terme générique d'un grand nombre de ressources numériques qui utilisent des techniques de chiffrement pour générer des unités de monnaie, vérifier les transactions, et transférer de la valeur. Généralement, les cryptomonnaies sont considérées décentralisées. Une cryptomonnaie ne doit pas être confondue avec une monnaie virtuelle qui est un type de monnaie numérique qui est habituellement contrôlée par ses créateurs et développeurs. Quelques exemples de monnaies virtuelles sont gametime dans World of Warcraft, ROBUX dans Roblox, les programmes de points de récompense, ou Ripple, tout cela peut être échangé contre de la monnaie ou des valeur marchandes, mais elles ne sont pas considérés comme des cryptomonnaies car elles sont centralisées, contrôlées et émises par une entité unique.
 
-Monero is one of many cryptocurrencies currently available. Other examples are Bitcoin, Litecoin, Dogecoin, Dash, Zcash, etc, but nearly all other cryptocurrencies lack features that make them a true money (most importantly @fungibility which is a requirement for it to be a store-of-value).
+Monero est l'une des nombreuses cryptomonnaies actuellement disponible. D'autres exempls sont Bitcoin, Litecoin, Dogecoin, Dash, Zcash, etc. mais la plupart des autres cryptomonnaies sont exempte de fonctionnalités qui feraient d'elles de vrai monnaies (la plus importante étant la @fongibilité qui est un prérequis pour qu'elle soit une réserve de valeur).
 
-Not all cryptocurrencies operate the same, but they usually share the properties of decentralization, encryption, and the ability to send and receive transactions. Most are irreversible, pseudonymous, global, and permissionless. Most aim to be a store-of-value or be digital cash that allows you to transact.
+Toutes les cryptomonnaies ne fonctionnent pas de la même manière, mais elles partagent généralement des propriétés de décentralisation, chiffrement, et la capacité à envoyer et recevoir des transactions. La plus part sont irrévocables, pseudonymiques, globales, et libre d'utilisation. La plus part ont pour objectif d'être des réserves de valeurs pour de la monnaie numérique avec laquelle vous pouvez effectuer des transactions.
```

Comment:
valeurs pour de la --> valeurs ou la ... ?

Reply:
ok. and "plus part" => "plupart"